evap/evaluation/models.py

```diff
@@ -512,19 +520,35 @@ class TextAnswer(Answer):
     """A free-form text answer to a question (usually a comment about a course
     or a contributor)."""
 
-    elements_per_page = 5
-
     reviewed_answer = models.TextField(verbose_name=_("reviewed answer"), blank=True, null=True)
     original_answer = models.TextField(verbose_name=_("original answer"), blank=True)
-
-    checked = models.BooleanField(verbose_name=_("answer checked"), default=False)
-    hidden = models.BooleanField(verbose_name=_("hide answer"), default=False)
+    
+    TEXT_ANSWER_STATES = (
+        ('N', _('not published')),
```

Comment:
I would prefer more descriptive (i.e. longer) names for the different states instead of `'N'`, `'Y'`, `'P'` and `''`. Especially because it is easy to mix up `'Y'` and `'P'`.

I don't know whether the the (translated) state names are appropriate, since I am not sure whether I understand how all the different states should work. Is this already documented somewhere? Especially, what is is the definition of `published privately` (note the oxymoron) and how exactly is it enforced to be private?